### DISCUSSION TITLE
registries should be able to provide triplets in addition to packages

### DISCUSSION THREAD

**Discussion by justusranvier** [microsoft/vcpkg#31313] [Ideas]
As far as I can tell from the [documentation for registries](https://learn.microsoft.com/en-us/vcpkg/users/registries) and my own experimentation, registries can only provide packages while overlays can provide both packages and triplets.

This means if you're using overlays which provide triplets and you want to switch to a registry you can only move the packages to the registry and still need to keep the overlays around.

It would be more convenient if vcpkg-configuration.json could contain a "triplets" array with similar functionality to "packages"

**Comment by esdandreu** [comment]
I agree. I've been looking for a way to know where does vcpkg fetch the custom registries in order to set overlay triplets to the same folder.
